src/tweak.cpp

```diff
@@ -108,7 +107,8 @@ UniValue settweak(const UniValue& params, bool fHelp)
     {
         throw runtime_error(
             "set"
```

Comment:
why `set` is on a new line whereas a few line above (L60) get was moved on the same line with `throw runtime_error(`? 

It seems to me that they should be formatted the same way. (slightly prefer the `set` formatting, but I don,t mind to much)